docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -321,6 +321,10 @@ In Dagit, you can manually trigger a test evaluation of a sensor and view the re
 
    If the run was successful, then for each produced run request, you can open the launchpad pre-scaffolded with the config produced by that run request. You'll also see a new computed cursor value from the evaluation, with the option to persist the value.
 
+<Note>
+  Sensors constructed with the `@asset_sensor`, `@run_status_sensor`, and `@run_failure_sensor` decorators cannot be tested using this button as of yet.
+</Note>
```

Comment:
Thoughts on combining this with the `Note` component that already exists on 286? That way, users can see which sensors aren't supported before they try to step through the process.

I wrote up and tested this out locally - it could be a copy/paste of the existing component:

```
<Note>
  <strong>Before you test</strong>, note that:
  <ul>
    <li>
      Test evaluations of sensors run the sensor's underlying Python function,
      meaning that any side effects contained within that sensor's function may
      be executed.
    </li>
    <li>
      <strong>This method doesn't support all sensor types.</strong> Currently,
      the <strong>Test Sensor</strong> button doesn't support sensors
      constructed using the following decorators:{" "}
      <PyObject object="asset_sensor" decorator />,{" "}
      <PyObject object="run_status_sensor" decorator />, or{" "}
      <PyObject object="run_failure_sensor" decorator />.
    </li>
  </ul>
</Note>
```

(Also, we can't currently use Markdown inside components like `Note` - it won't render correctly)